tests/salmon_tests/__init__.py

```diff
@@ -1,17 +1,24 @@
 import os
 import shutil
+import subprocess
+import time
 
 import config.testing  # noqa
 
 
 dirs = ["run", "logs"]
+DUMMY_RELAY=None
 
 
 def setup_package():
     for path in dirs:
         os.mkdir(path)
+    DUMMY_RELAY=subprocess.Popen("python -m smtpd -n -c DebuggingServer localhost:7899 > logs/smtpd.log", shell=True)
+    time.sleep(10)
```

Comment:
That's an old comment, it should be removed ☺

As far as I can see, all the tests that fail without a test server are all testing **sending**, not receiving. 

You can use mock like tihs:
```
@patch("salmon.server.smtplib.SMTP")
def test_server_test(client_mock):
    # test code
```

This is the documentation for the mock library we're using: http://www.voidspace.org.uk/python/mock/